src/app/tests/suites/certification/Test_TC_DISHM_1_1.yaml

```diff
@@ -0,0 +1,152 @@
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision from DUT"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads the FeatureMap from DUT"
+      PICS: " !DISHM.S.F00 && !DISHM.S.F01 && !DISHM.S.F02 "
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label: "Given DISHM.S.F00(DEPONOFF) ensure featuremap has the correct bit set"
+      PICS: DISHM.S.F00
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 1
+          constraints:
+              type: bitmap32
+
+    - label: "Given DISHM.S.F01(EXT) ensure featuremap has the correct bit set"
+      PICS: DISHM.S.F01
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 1
+          constraints:
+              type: bitmap32
+
+    - label: "Given DISHM.S.F02(DEPCTM) ensure featuremap has the correct bit set"
+      PICS: DISHM.S.F02
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 1
+          constraints:
+              type: bitmap32
+
+    - label: "TH reads AttributeList from DUT"
+      PICS: DISHM.S.Afffb
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      0,
+                      1,
+                      2,
+                      3,
+                      65528,
+                      65529,
+                      65530,
+                      65531,
+                      65532,
+                      65533,
+                  ]
+
+    - label: "TH reads optional attribute (DISHM.S.A0004(StartUpMode)) in AttributeList"
+      PICS: DISHM.S.A0004
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [4]
+
+    - label: "TH reads optional attribute (DISHM.S.F00(DEPONOFF)) in AttributeList"
+      PICS: DISHM.S.F00
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [5]
+
+    - label: "TH reads AcceptedCommandList from DUT"
+      PICS: "!DISHM.S.F02"
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [0]
+
+    - label: "TH reads AcceptedCommandList from DUT"
+      PICS: DISHM.S.F01
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [1]
```

Comment:
This seems to be based on an old version of the spec and of the test plan?  Please double-check this against the test plan carefully...  There is no command 1 in this cluster.

I'm going to stop here for now, until those checks are done.